### PR TITLE
Parametrize tests for transformation functions

### PR DESCRIPTION
**Pull request by jakobchwastek**
## Description

Merged the single tests for `transforms.pos2vel` and `transforms.deg2vel` into one test by pytest parametrization:

## Implemented changes
- `test_pix2deg_raise_error` function for `pix2deg` raising error `expected_error` with input `kwargs`
- `test_pos2vel_raise_error` function for `pos2vel` raising error `expected_error` with input `kwargs`
- `test_pix2deg_equals` to check if `pix2deg` returns `expected_value` with input `kwargs` 
- `test_pos2vel_returns` to check if `pos2vel` returns `expected_value` with input `kwargs` and calculation method `method`

### CODE REVIEW

**Comment by jakobchwastek** [comment]
Should we set a similar name for return tests `test_pix2deg_equals` and `test_pos2vel_returns` ?

**Comment by dkrako** [comment]
> Should we set a similar name for return tests `test_pix2deg_equals` and `test_pos2vel_returns` ?

Yes, good idea.
Let's name it `test_pix2deg_returns` and `test_pos2vel_returns` then.

Also rename the functions which test for exception raising:
`test_pix2deg_raise_error` to `test_pix2deg_raises_error` and the same for the one for `pos2vel`.

There are three small flake8 errors left.

Otherwise, this looks nice and can be merged after these small fixes.

File: tests/test_transformations.py [hunk 1]
```diff
@@ -7,262 +7,437 @@
...
+            id='linear_input_returns_constant_velocity'
+        )
+    ]
+)
+def test_pos2vel_returns(method, kwargs, padding, expected_value):
+    actual_value = pos2vel(**kwargs, method=method)
+    assert (actual_value[padding[0]:padding[1]] == expected_value[padding[0]:padding[1]]).all()
 
 
 def test_pos2vel_stepped_input_returns_alternating_velocity_method_preceding():
```
Comment (dkrako) [inline tests/test_transformations.py:443]: one last thing:
can you parametrize these three functions too?

just call it `test_pos2vel_stepped_input_returns`

leave the creation of `N` and `x` in the function and have params like this:

```
@pytest.mark.parametrize(
    'method, expected',
    [
        pytest.param(
            'preceding', np.array([2.0, 0.0] * (N // 2)),
            id='method_preceding_alternating_velocity',
        ),
```

**Review by dkrako — CHANGES_REQUESTED** [review]
the refactoring of the last three methods is remaining

File: tests/test_transformations.py (hunk 1 above)
Comment (SiQube) [reply tests/test_transformations.py]: @jakobchwastek @dkrako is this resolved and mergeable?

Comment (dkrako) [reply tests/test_transformations.py]: Although this is not resolved yet, I decided to merge this PR.
I will create another issue for this remaining task.

**Review by dkrako — APPROVED** [review]
Although this is not resolved yet, I decided to merge this PR.
I will create another issue for this remaining task.

**Comment by dkrako** [comment]
Ah sorry, I see this was already fixed in november and I just didn't get it.
Must have been blind or so.

**Pull request merged**